docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Use Codewind in CodeReady Workspaces to build high quality cloud native applications."
+permalink: codewind-crw-quick-guide.html
+duration: 15 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
+## Objectives
+* Install CodeReady Workspaces and Codewind
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local machine.  
+
+### Prerequisite
+CodeReady Workspaces require at least two 5Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 5Gi RWO volume for each created workspace.
+
+Each Codewind workspace also requires at least on 5Gi ReadWriteMany (RWX) persistent volume.
+
+Before you can use Codewind with CodeReady Workspaces, apply the Codewind cluster role with `oc apply -f`. 
+
+### Installing CodeReady Workspaces
+1\. Log in to your OpenShift cluster. 
+
+2\. From the sidebar, click **Operators** then **OperatorHub**.
+
+3\. Search for `CodeReady Workspaces`. 
+
+4\. Select **Red Hat CodeReady Workspaces** and click **Install**: 
+
+![Subscribe to OperatorHub](./images/codereadyworkspaces-images/CRW-OperatorHub.png){:width="900"}. 
+
+![Install CodeReady Workspaces](./images/codereadyworkspaces-images/CRW-Install.png){:width="800"}. 
```

Comment:
This, the screenshot, and the optional configurations should be below the sample yaml